install_config/cluster_metrics.adoc

```diff
@@ -600,25 +600,38 @@ To avoid this issue, use certificates which are configured to be acceptable by
 your browser.
 ====
 
-ifdef::openshift-origin[]
 == Accessing Hawkular Metrics Directly
 
 To access and manage metrics more directly, use the Hawkular Metrics API.
 
+[NOTE]
+====
+When accessing Hawkular Metrics via the API you will only be able to perform
+reads, writing metrics has been disabled by default.  If you wish for
```

Comment:
Maybe we should mention this (single spaces between sentences) in
`contributing_to_docs/doc_guidelines.adoc`?  I looked for this specifically
before opening the PR but didn't find anything.

Of course now I realize the file itself uses that style =)